rfcs/013-release_deployment_tracking/v1/README.md

```diff
@@ -0,0 +1,181 @@
+# Version 1 
+
+## Implementation from 24/01/2019
+
+This solution describes a set of steps to track build, release and deployment but is deliberately agnostic of deployment mechanism. 
+
+We make use of SSM to track which release should deploy into which environment. Out terraform stacks read from these SSM parameters to get which images to apply.
```

Comment:
nit: "We make use of" ~> "We use"
nit: "should deploy" ~> "should be deployed"
nit: "Out terraform stacks read from these SSM parameters" ~> "Our Terraform code reads from SSM"